src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,64 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.StandardOpenOption;
+import java.util.ArrayList;
+import java.util.List;
+
 public class WorkWithFile {
+    private static final String COMMA = ",";
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final String RESULT = "result";
+    private static final int INDEX_OF_OPERATION = 0;
+    private int totalSupply = 0;
+    private int totalBuy = 0;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        File file = new File(toFileName);
+        List<String> list = createReport(readFromFile(fromFileName));
+
+        try {
+            file.createNewFile();
+        } catch (IOException e) {
+            throw new RuntimeException("Can't create new file", e);
+        }
+
+        try {
+            for (String row : list) {
+                Files.write(file.toPath(), row.getBytes(), StandardOpenOption.APPEND);
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write to file " + toFileName, e);
+        }
```

Comment:
let's move this logic into private `writeToFile(String fileName, String content)` method